plasma_framework/test/src/exits/payment/PaymentInFlightExitable.test.js

```diff
@@ -166,16 +167,17 @@ contract('PaymentInFlightExitable', ([_, alice, bob, carol]) => {
                 const exitId = await this.exitIdHelper.getInFlightExitId(this.args.inFlightTx);
 
                 const exit = await this.exitGame.inFlightExits(exitId);
+
                 expect(exit.bondOwner).to.equal(alice);
                 expect(exit.oldestCompetitor).to.be.bignumber.equal(new BN(0));
-                expect(exit.exitStartTimestamp).to.be.bignumber.equal(ethBlockTime);
+                // expect(exit.exitStartTimestamp).to.be.bignumber.equal(ethBlockTime); FIXME: why this fails?
```

Comment:
sigh, so this is back. Basically ganache would keep on increasing the time even no new block mined. So when your tests run slower then certain threshold this `equal` statement would break.

We could probably do the same thing as solidity helper tests do `to.be.bignumber.closeTo(...)`:
https://github.com/OpenZeppelin/openzeppelin-test-helpers/blob/master/test/src/time.test.js#L68

Reply:
thanks, done